app/_hub/kong-inc/aws-lambda/index.md

```diff
@@ -66,127 +55,173 @@ params:
   config:
     - name: aws_key
       required: semi
-      value_in_examples: AWS_KEY
+      value_in_examples: <AWS_KEY>
       urlencode_in_examples: true
       default:
-      description: The AWS key credential to be used when invoking the function. This value is required if `aws_secret` is defined. If `aws_key` and `aws_secret` are not set, the plugin uses an IAM role inherited from the instance running Kong to authenticate.
+      datatype: string (encrypted)
+      description: |
+        The AWS key credential to be used when invoking the function. The `aws_key` value is required
+        if `aws_secret` is defined. If `aws_key` and `aws_secret` are not set, the plugin uses an
+        IAM role inherited from the instance running Kong to authenticate.
```

Comment:
```suggestion
        IAM role inherited from the instance running Kong to authenticate. Can be symmetrically encrypted, if using Kong Enterprise and if configured: https://docs.konghq.com/enterprise/latest/db-encryption/
```

Reply:
@tlblessing actually, do you know how to not hardcode the `Kong Enterprise` part? I think there's a template somewhere like this? `{ product_name }`? I think the name `Kong Enterprise` is not accurate anymore.

> This is the product previously referred to as Kong Gateway Enterprise.  In version 2.3, we released a free operating mode of Kong Gateway Enterprise, and given it no longer needs a paid “Enterprise” license, we now refer to this gateway as the Kong Gateway and disambiguate from the OSS-only Gateway as Kong Gateway (OSS).
>
> https://konghq.com/blog/kong-gateway-2-4/